docs/content/en/docs/reference/baremetal/bare-custom-hookos.md

```diff
@@ -3,6 +3,168 @@ title: "Customize HookOS for EKS Anywhere on Bare Metal"
+     - linuxkit/init:v0.8
+    @@ -42,7 +42,7 @@ services:
+     binds:
+     - /var/run:/var/run
+     - name: docker
+    - image: quay.io/tinkerbell/hook-docker:0.0 (http://quay.io/tinkerbell/hook-docker:0.0)
+    + image: localhost:5000/tinkerbell/hook-docker:0.0
+     capabilities:
+     - all
+     net: host
+    @@ -64,7 +64,7 @@ services:
+     - /var/run/docker
+     - /var/run/worker
+     - name: bootkit
+    - image: quay.io/tinkerbell/hook-bootkit:0.0 (http://quay.io/tinkerbell/hook-bootkit:0.0)
+    + image: localhost:5000/tinkerbell/hook-bootkit:0.0
+     capabilities:
+     - all
+    ```
+
+    The changes above are for using local registry (localhost:5000) for hook-docker, hook-bootkit, and hook-kernel. 
+
+    >**_NOTE_**: You may also need to modify the `hook.yaml` file if you want to add or change components that are used to build up the image. So far, for example, we have needed to change versions of `init` and `getty` and inject SSH keys. Take a look at the [LinuxKit Examples](https://github.com/linuxkit/linuxkit/tree/master/examples) site for examples.
+    >
+
+1. Make any planned custom modifications to the files under `hook`, if you are only making changes to `bootkit` or `tink-docker`.
+    
+    
+1. If you are modifying the kernel, such as to change kernel config parameters to add or modify drivers, follow these steps:
+
+    * Change into kernel directory and make a local image for amd64 architecture:
+
+    ```bash
+    cd kernel; make kconfig_amd64
+    ```
+
+    * Run the image
+
+    ```bash
+    docker run --rm -ti -v $(pwd):/src:z quay.io/tinkerbell/kconfig
+    ```
+
+    * You can now navigate to the source code and run the UI for configuring the kernel:
+
+    ```bash
+    cd linux-5-10
+    make menuconfig
+    ```
+
+    * Once you have changed the necessary kernel configuration parameters, copy the new configuration:
+
+    ```bash
+    cp .config /src/config-5.10.x-x86_64
+    ```
+
+    Exit out of container into the repo’s kernel directory and run make:
+
+    ```bash
+    /linux-5.10.85 # exit
+    user1 % make
+    ```
+
+1. Install Linuxkit based on instructions from the [LinuxKit](https://github.com/linuxkit/linuxkit) page.
+    
+    
+1. Ensure that the `linuxkit` tool is in your PATH:
+
+    ```bash
+    export PATH=$PATH:/home/tink/linuxkit/bin
+    ```
+
+1. Start a local registry:
+
+    ```bash
+    docker run -d -p 5000:5000 -—name *registry* *registry*:2
+    ```
+
+1. Compile by running the following in the root of the repo:
+
+    ```bash
+    make dist  
+    ```
+1. Artifacts will be put under the `dist` directory in the repo’s root:
+
+    ```bash
+    ./initramfs-aarch64
+    ./initramfs-x86_64
+    ./vmlinuz-aarch64
+    ./vmlinuz-x86_64
+    ```
+
+1. To use the kernel (`vmlinuz`) and initial ram disk (`initramfs`) when you build your cluster, see the description of the `hookImagesURLPath` field in your [Bare Metal configuration]({{< relref "../clusterspec/baremetal/" >}}) file.
```

Comment:
We should point this link to the exact anchor for the hookImagesURLPath field description

Reply:
Done.